### ISSUE TITLE
Labeling windows

### ISSUE THREAD

**Issue by jbriales** [koekeishiya/yabai#2383]
I checked the documentation thoroughly and it seems there is no support for labeling windows.
I.e. I'd expect sth like: `yabai -m window [<WINDOW_SEL>] --label LABEL`.

This would be useful to quickly tag certain windows of interest directly from within yabai, w/o messing with the title.

Any thoughts or alternatives on this?

**Comment by jbriales** [comment]
Hi! Gentle ping on this!
As we enter the weekend, I wondered if I should have a look at the internals to try and prototype this myself or if this is not available by design for some reason :)

**Comment by koekeishiya** [comment]
I didn't find it particularly useful after adding scratchpad windows.

**Comment by jbriales** [comment]
I agree it overlaps with scratchpad but I might want to label windows that are not in the scratchpad (not floating, etc.).
Would you be against having the additional labeling capability?
